backend/package-lock.json

```diff
@@ -1,3813 +1,6 @@
 {
```

Comment:
Please update your Node and npm versions! Lmk if you need any help getting that done.